applications/GeoMechanicsApplication/tests/test_element_lab/oedometer_ULFEM/README.md

```diff
@@ -0,0 +1,21 @@
+# Oedometer ULFEM test
+This test simulates an oedometer lab test. The UPwUpdatedLagrangianElement2D3N element is used to simulate the soil sample in a very simple 4-noded mesh, as displayed in the following figure:
+
+![MeshStructure.png](MeshStructure.png)
+
+## Setup
+The test is performed in a single stage, with the following conditions:
+- Constraints:
+  - The X and Y displacement in the bottom two nodes (3 and 4) are fixed to 0.0.
+  - The X displacement in all nodes is fixed to 0.0.
+- Material:
+    - The material is described using a linear elastic material with a GeoLinearElasticPlaneStrain2DLaw.
+- Conditions:
+  - A UPwFaceLoadCondition2D2N is added to the top side of the cube (nodes 1 and 2).
+- Loads: 
+  - A line load in the y direction is applied to the top side of the cube (nodes 1 and 2), linearly ramping up from 0 to -1000000 kPa (?) in the time interval \[0, 1\].
```

Comment:
Part of this setup could be visualized instead of having text. @WPK4FEM, @mcgicjn2 Let me know if you'd prefer that as well, or text like this is more clear.